src/types/ajv/BroadcastFinalStateReq.ts

```diff
@@ -0,0 +1,31 @@
+import { addSchema, addSchemaDependency } from '../../utils/serialization/SchemaHelpers'
+import { AJVSchemaEnum } from '../enum/AJVSchemaEnum'
+
+const BroadcastFinalStateReqSchema = {
+    type: 'object',
+    properties: {
+        txid: { type: 'string' },
+        stateList: {
+        type: 'array',
+        items: { $ref: AJVSchemaEnum.WrappedDataResponse },
```

Comment:
Please use the schema here instead of referencing the enum. Do check the Ajv Schemas for schemaBroadcastStateReq and schemaRequestStateForTxPostResp

Reply:
```javascript
import { addSchema } from '../../utils/serialization/SchemaHelpers'
import { schemaWrappedData } from './WrappedData'

export const schemaGetAccountDataByListResp = {
  properties: {
    accountData: {
      type: ['array', 'null'],
      items: schemaWrappedData,
    },
  },
  required: ['accountData'],
}
```

Here's an example of the approach @abdulazeem-tk4vr is requesting changes for. $ref do not work.

Also `AJVSchemEnum` does not hold actual schema they're only for namespace identifier for each ajv types. It is NOT actual schema definition. Please look at one of ajv migration that are already done.